### ISSUE TITLE
Doesn't work if the tenant is set to create groups in the teams url

### ISSUE THREAD

**Issue by ozippy** [SharePoint/sp-starter-kit#30]
#### Category
- [ ] Question
- [x ] Bug
- [ ] Enhancement

I am trying to install on my tenant. Targeted release is enabled for all. I have tried many times over the past few days to get it to deploy, but with limited success.

Firstly I have had to turn off multi-factor auth for my account as I could not get it to work at all.

It took me ages to realise that my tenant was set to create new groups under the /teams/.. URL which was causing the deploy script to fail with a lot of errors because it assumes a path of /sites/... 

I have been able to get it to partially deploy. However the theme doesn't get applied. I am not sure if this is because of the previous failures.

I love what you are doing with this initiative, it looks really useful.

I also noticed that the Edit button for My Links doesn't seem to work on my iPad in Chrome. 

Thanks

Jeremy

This is the most recent output from the deploy script;

Security warning
Run only scripts that you trust. While scripts from the internet can be useful, this script can potentially harm your
computer. If you trust this script, use the Unblock-File cmdlet to allow the script to run without this warning
message. Do you want to run C:\Github\sp-starter-kit\sp-starter-kit-master\provisioning\deploy.ps1?
[D] Do not run  [R] Run once  [S] Suspend  [?] Help (default is "D"): r

Security warning
Run only scripts that you trust. While scripts from the internet can be useful, this script can potentially harm your
computer. If you trust this script, use the Unblock-File cmdlet to allow the script to run without this warning
message. Do you want to run C:\Github\sp-starter-kit\sp-starter-kit-master\provisioning\functions.ps1?
[D] Do not run  [R] Run once  [S] Suspend  [?] Help (default is "D"): r
Reading Site Hierarchy from ./hierarchy.json
Creating https://sanitariumdev.sharepoint.com/sites/group6portal
Creating group6hr
Creating group6marketing
Provisioning solution
Update-AppIfPresent : Cannot process argument transformation on parameter 'Connection'. Cannot convert the
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection" value of type
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection" to type
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection".
At C:\Github\sp-starter-kit\sp-starter-kit-master\provisioning\deploy.ps1:96 char:92
+ ... "sharepoint-starter-kit-client-side-solution" -Connection $connection
+                                                               ~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Update-AppIfPresent], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Update-AppIfPresent

Set-ThemeIfNotSet : Cannot process argument transformation on parameter 'Connection'. Cannot convert the
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection" value of type
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection" to type
"SharePointPnP.PowerShell.Commands.Base.SPOnlineConnection".
At C:\Github\sp-starter-kit\sp-starter-kit-master\provisioning\deploy.ps1:106 char:31
+ Set-ThemeIfNotSet -Connection $connection
+                               ~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Set-ThemeIfNotSet], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Set-ThemeIfNotSet

Applying template to portal
WARNING: You specified to remove existing content types for the list with url '/sites/group6portal/Lists/Alerts'. We
found a list with the same url in the site. In case of a list update we cannot remove existing content types as they
can be in use by existing list items and/or documents.

Id                                   Title                      Site Scripts
--                                   -----                      ------------
efe32d78-5513-44e9-93b1-0a2703760672 Contoso Team Site          {e22d4aae-71ca-41a8-be5a-435a1bfaec52}
285de1dc-1bb7-4953-a08e-5182fdc2dbd2 Contoso Communication Site {6a0c4682-4478-40b8-8ba3-5ee3866a57d3}
Updating navigation and applying collab templates
Done.

PS C:\Github\sp-starter-kit\sp-starter-kit-master\provisioning>

**Comment by bcameron1231** [comment]
This looks like a duplicate of #12 . Maybe these can be consolidated.

**Comment by bcameron1231** [comment]
Closing this, as it is a duplicate.
